vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -28,6 +28,7 @@ def call(Map pipelineParams) {
                     0 8 * * 6 %scylla_version=master:latest;labels_selector=alternator-weekly;requested_by_user=radoslawcybulski
                     00 6 * * 0 %scylla_version=master:latest;labels_selector=master-weekly;requested_by_user=juliayakovlev
                     0 23 */21 * * %scylla_version=master:latest;labels_selector=master-3weeks;requested_by_user=juliayakovlev
+                    13 6 8-14 * 2 %scylla_version=master:latest;labels_selector=gce-custom-monthly;requested_by_user=valerii.ponomarov
```

Comment:
I don't remember that this is part of what we talked, let's raise a task to revise it in couple of months, if it's needed

Reply:
we agreed to leave for now